### DISCUSSION TITLE
1.5.2

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#588] [Обновления]
# Список изменений
- Исправлена работа функции "связать громкость перевода и видео" через слайдер ютуба (#535)
- Исправлена ошибка из-за которой слайдер ютуба не обновлялся, если изменялось значение громкости перевода в меню с включенной функцией "связать громкость перевода и видео"
- Исправлена ошибка из-за которой не менялось значение слайдера громкости оригинала, если видео мутилось через кнопку или хоткей
- Немного оптимизирована работа с syncVolumeObserver. Теперь, он проверяет изменения, только, если включен перевод видео и включена функция "связать громкость перевода и видео"
- Добавлена двухсторонняя синхронизация слайдера громкости в меню со слайдером громкости плеера. Поддерживаемые плееры: Стандартный, VK, Vimeo, YouTube (уже была поддержка), Trovo, YandexDisk, возможно, какие-то другие - не указанные в этом списке
- Включена возможность использования функции "связать громкость перевода и видео" для всех поддерживаемых сайтов
- Убрана обводка слайдеров при нажатие на слайдер в Vimeo
- Убрана жирность текста в меню на Rutube
- Исправлено отображение субтитров в ОК (#561)
- Добавлен `peertube.su` в список доступных доменов для Peertube (#442)
- Добавлена возможность перемещения кнопки перевода по аналогии с Яндекс Браузером (#483). Работает, только, если видео более 550 пикселей
- Добавлено перемещение меню в зависимости от позиции кнопки. Работает, только, если видео более 550 пикселей. Тестировалось только на YouTube, Vimeo, Rutube, VK и прямых ссылках на видео файлы
- Большинство важных fetch заменено на GM_fetch, благодаря чему, должно уменьшиться число ошибок из-за CSP

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.5.1.5...1.5.2

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.2'>1.5.2</a>.</em>

**Comment by NullVerdict** [comment]
А вот и первые баги, нужно добавить домен в connect чтобы он работал.
![изображение](https://github.com/ilyhalight/voice-over-translation/assets/24498484/ffeeeeec-aa34-448c-9c85-68cf521bbe22)
![изображение](https://github.com/ilyhalight/voice-over-translation/assets/24498484/0c5e19d3-8d74-44c5-80b3-d4643c970600)

**Reply by ilyhalight** [reply]
исправил
